services/web/server/src/simcore_service_webserver/projects/_trash_api.py

```diff
@@ -56,8 +57,8 @@ async def _is_project_running(
             app, user_id=user_id, project_id=project_id
         )
     ) or bool(
-        await director_v2_api.list_dynamic_services(
-            app, user_id=user_id, project_id=f"{project_id}"
+        await dynamic_scheduler_api.list_dynamic_services(
```

Comment:
Can `list_dynamic_services` raise differently to the old one? i.e. can you document a bit in the function some known exceptions?

Reply:
It raises different errors for sure. But does not raise any extra known errors.